packages/client/hmi-client/src/components/workflow/tera-create-workflow-modal.vue

```diff
@@ -74,15 +82,24 @@ const scenarios = ref<ScenarioItem[]>([
 		id: SensitivityAnalysisScenario.templateId,
 		instance: new SensitivityAnalysisScenario(),
 		component: markRaw(TeraSensitivityAnalysisTemplate)
+	},
+	{
+		displayName: DecisionMakingScenario.templateName,
+		id: DecisionMakingScenario.templateId,
+		instance: new DecisionMakingScenario(),
+		component: markRaw(TeraDecisionMakingTemplate)
 	}
 ]);
```

Comment:
I think this scenarios variable doesn't need to be a ref. Also I suggest you make these accessible by an `id` key or something so you need to do a `find` in `getScenario`, you can just access the component you want by the key.

Reply:
now using a map!